src/fable-library/TimeSpan.ts

```diff
@@ -76,7 +76,8 @@ export function totalMinutes(ts: number) {
 }
 
 export function totalSeconds(ts: number) {
-  return ts / 1000;
+  // Round needed to replicated .Net behaviour
+  return Math.round(ts / 1000 * 1000000) / 1000000;
```

Comment:
Will this work? At the end you're dividing `ts` by 1000000000000000. And it seems there's no proper test for `.TotalSeconds`. Do you have a link to the .Net source that it's doing this operation?

Reply:
Ups, my memory of association rules is pretty broken, this is `(ts / 1000) * 1000000` 😅

Reply:
I guess you multiply by `1000000` to convert to ticks before the rounding. But if we're going to do this it's probably better to just use ticks instead of milliseconds as Fable is currently doing, and we should use longs because we risk overflowing JS numbers when converting to ticks. Also, this PR may fix `totalSeconds` but we still have `totalMinutes`, `totalHours`, etc...

I think we should do the rounding when creating the TimeSpan and there are two options:

- To be closer to .NET we can use a long to represent ticks (or just create a `TimeSpan` class).
- Keep using milliseconds if we don't mind losing a bit of precision for performance and simplicity but doing the rounding creating the timespan, here: https://github.com/fable-compiler/Fable/blob/3e544d67749f5c91d4ddf0ec89b40571468f8f26/src/fable-library/TimeSpan.ts#L19

Reply:
The goal of the formula is to round+truncate the value.

<img width="323" alt="Capture d’écran 2019-07-30 à 10 34 17" src="https://user-images.githubusercontent.com/4760796/62113823-9c24e880-b2b5-11e9-8e42-1d2eaf9fbfbd.png">

Reply:
Ooops, you had answered me since I opened the tabs ^^

In the end, I removed my change and added tests for all the `TotalXXX` methods to check that the float representation of the value is correctly handed inside the same runtime.

It's possible that later we have to use `Long` as you said if we want to have the same precision as in .Net but for me, it's outside the scope of this PR.